kubernetes/provider.go

```diff
@@ -27,6 +28,8 @@ import (
 	aggregator "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset"
 )
 
+const defaultFieldManagerName = "Terraform"
```

Comment:
we should probably have a module at the top level for things like this that are shared between the main provider and manifest